docs/examples/frameworks/mxnet/mxnet-various-readers.ipynb

```diff
@@ -96,31 +96,26 @@
    "metadata": {},
```

Comment:
<blockquote>common part of pipeline, other pipelines will inherit it</blockquote><ul><li>There's no inheritance</li><li>There's no pipeline yet (as it used to be), so "other pipelines" doesn't make sense any more</li></ul><p> </p><p>Suggested alternative:</p><p> </p><p>common part of <strong>the processing graph</strong>, used by all pipelines</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done</p>